packages/bpk-component-barchart/src/BpkBarchartBars.js

```diff
@@ -42,7 +44,31 @@ const getBarHeight = (
 const isOutlier = (point, { yScaleDataKey, maxYValue }) =>
   point[yScaleDataKey] > maxYValue;
 
-const BpkBarchartBars = props => {
+type Props = {
+  data: Array<any>,
+  xScaleDataKey: string,
+  yScaleDataKey: string,
+  height: number,
+  xScale: Function,
+  yScale: Function,
```

Comment:
We typically try to avoid `Function` in favour of more specific syntax like `(argType) => returnType`, is there something more specific we can use here?